Formula/v2ray-plugin.rb

```diff
@@ -0,0 +1,17 @@
+class V2rayPlugin < Formula
+  desc "SIP003 plugin based on v2ray for shadowsocks"
+  homepage "https://github.com/shadowsocks/v2ray-plugin"
+  url "https://github.com/shadowsocks/v2ray-plugin/archive/v1.0.tar.gz"
+  sha256 "5320eff71d99edcfa1e5e883debb42c50ccdebc2b43eb30c6c1b16baa632cda2"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"v2ray-plugin"
+  end
+
+  test do
+    output = shell_output("#{bin}/v2ray-plugin -help 2>&1", 2)
```

Comment:
Would it be possible to write a test that uses some functionality of the software rather than just printing text ?

Reply:
Hi, I've updated the commit.

Reply:
Oh, sorry for the misunderstand. It's a little complicated to test the functionality, but I'm working on it.